rosbag2_converter_default_plugins/src/rosbag2_converter_default_plugins/cdr/cdr_converter.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018, Bosch Software Innovations GmbH.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "cdr_converter.hpp"
+
+#include <memory>
+#include <string>
+
+#include "rcutils/strdup.h"
+#include "rmw/rmw.h"
+#include "../logging.hpp"
+
+namespace rosbag2_converter_default_plugins
+{
+
+void CdrConverter::deserialize(
```

Comment:
I'd prefer a similar argument order than in the `rmw_deserialize` function:
https://github.com/ros2/rmw/blob/master/rmw/include/rmw/rmw.h#L247-L253

the `ros_message` is the last argument, so being consistent with the place of the input/output arguments.

Reply:
Makes perfect sense, will do.